### ISSUE TITLE
K8s integration: Include resource utilization in default scrape configuration

### ISSUE THREAD

**Issue by nickbp** [opstrace/opstrace#1070]
The default scrape configuration provided by the K8s integration currently covers `/metrics` on the kubelets, but not `/metrics/cadvisor`, which contains useful resource utilization metrics like `container_cpu_usage_seconds_total`. It'd make sense to include this in the default scrape config.

So far, I've successfully scraped `/metrics/cadvisor` from a test onprem K3s cluster using the following addition:
```
    # Collection of per-node metrics
    - job_name: 'kubernetes-nodes-cadvisor'
      kubernetes_sd_configs:
      - role: node

      metrics_path: /metrics/cadvisor
      scheme: https
      # TLS config for getting list of nodes to scrape, not querying nodes themselves
      tls_config:
        ca_file: /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
      authorization:
        credentials_file: /var/run/secrets/kubernetes.io/serviceaccount/token

      relabel_configs:
      - target_label: __scheme__
        replacement: https
      # Include node hostname
      - source_labels: [__meta_kubernetes_node_label_kubernetes_io_hostname]
        target_label: instance
      # Include job label for the nodes
      - source_labels: []
        target_label: job
        replacement: kubelet-cadvisor
      # Include integration ID for autodetection in opstrace
      - source_labels: []
        target_label: integration_id
        replacement: 968a63b5-2b49-4ed9-b81e-818209cac34e
```

Need to check the following:
- I've only tried the above configuration on a test onprem cluster running K3s (added a stock integration then manually edited the `opstrace-prometheus` ConfigMap). Since the endpoint is against the kubelets themselves, should check some other clusters (e.g. EKS, GKE) and verify that the `/metrics/cadvisor` endpoint is also available there.
- The [K8s docs](https://kubernetes.io/docs/concepts/cluster-administration/system-metrics/#metrics-in-kubernetes) also mention `/metrics/resource` and `/metrics/probes` endpoints. While we're looking at this, can also check whether those endpoints have anything interesting.

The resulting integration changes should include the following:
- Updated ConfigMap template used for the K8s metrics integration to include the additional scrape target(s) - and tested across a few cluster environments to check there aren't any platform-specific kubelet issues.
- Updated prebuilt dashboards (provided via integration) to take advantage of the additional metrics (e.g. cpu/mem usage)

**Comment by nickbp** [comment]
Checked each of the three documented /metrics/* endpoints on a K3s kubelet:
- `/metrics/cadvisor`: 514KB, so the bulk of the metrics coverage seems to be here.
- `/metrics/probes`: 2911B, appears to just have stats on readinessProbes/livenessProbes for pods on that kubelet - specifically counts on how many successful/failed queries there were for each probe. Sort of a specific use-case, but doesn't hurt to include.
- `/metrics/resource`: 3230B, has a summary of memory and cpu usage for pods on the kubelet. appears to be a strict subset of what's available in `/metrics/cadvisor` but going to check

**Comment by nickbp** [comment]
Compared the content of `/metrics/resource` to `/metrics/cadvisor` and found the latter to MOSTLY be a superset in terms of available metrics and all their labels. However there are a couple metrics in `/metrics/resource` that don't have a direct match in `/metrics/cadvisor` and couldn't hurt to include. Here's an inventory of the metrics found in `/metrics/resource` and what equivalent can be found under `/metrics/cadvisor`:
- `container_cpu_usage_seconds_total` and `container_memory_working_set_bytes`: Pretty much duplicates across both. The cadvisor version has a superset of tags (with a few additional ones like `image`).
- `node_cpu_usage_seconds_total` and `node_memory_working_set_bytes`: Looks equivalent to cgroups' `container_cpu_usage_seconds_total{container="",id="/"}` and `container_memory_working_set_bytes{container="",id="/"}`. But that's not super intuitive..
- `pod_cpu_usage_seconds_total` and `pod_memory_working_set_bytes`: Similar to the above, this looks equivalent to cgroups' `container_cpu_usage_seconds_total{container=""}` and `container_memory_working_set_bytes{container=""}`

**Comment by nickbp** [comment]
The main difference for `/metrics/resource` and `/metrics/probes` is that all of the metrics are labeled as `[ALPHA]` in their descriptions, on a cluster running `v1.21.1+k3s1`, meanwhile `/metrics/cadvisor` does not have `[ALPHA]` anywhere. But that said, I checked older versions of the K8s documentation and they list all three of these endpoints going back as far as [K8s v1.17](https://v1-17.docs.kubernetes.io/docs/concepts/cluster-administration/monitoring/) (1.16 docs weren't loading with a 404 so I stopped searching). So it's not clear what `ALPHA` really means in this case, given that it's been documented for several K8s release cycles.

In any case, I don't think it hurts to include all three endpoints in our integration scrape config, since it looks like many clusters should support all of them - and worst case, the user always has the option of editing the scrape configmap we provide if they want to disable any of them.
